code/_onclick/item_attack.dm

```diff
@@ -10,6 +10,9 @@
 /obj/item/proc/tool_attack_chain(mob/user, atom/target)
 	if(!tool_behaviour)
 		return FALSE
+	if(SEND_SIGNAL(target, COMSIG_TOOL_ATTACK, src, user) & COMPONENT_CANCEL_TOOLACT)
```

Comment:
You should use the existing `COMPONENT_CANCEL_ATTACK_CHAIN` for this.

Reply:
I don't want to cancel the attack chain, though. I want it to continue through the attack chain but not act as a tool.

Reply:
Ah, that's fair! Makes sense to me.